### ISSUE TITLE
I cant play single draft mode

### ISSUE THREAD

**Issue by cesar6793** [ValveSoftware/Dota2-Gameplay#7006]
### Description

i have 2 low matches , every time i try to queue for single draft it said "Single draft is the only game mode allowed in the low priority punishmet poll" , i already unselected the resto of the modes and it keeps saying the same warning.

### Screenshots

![draftr](https://user-images.githubusercontent.com/13472848/219988157-ae27c396-2fca-4bf6-883c-1b18ae2111d6.png)

**Comment by TheUnholyWrath** [comment]
can you send a photo showing that you've actually unselected every other mode. every time this has been reported, that has been the issue

**Comment by PeruShouldBeKilled** [comment]
![preview](https://user-images.githubusercontent.com/125853304/220022481-4f29d0af-6630-4383-9c6b-55d5e35e8f17.png)

**Comment by commentator24** [comment]
just like #6079 #3644 you guys still have all pick mode or turbo selected under "unranked". must untick this.

**Comment by hansangwoo32** [comment]
I am having exactly same problem like this can someone hlep me ?
https://www.youtube.com/watch?v=jct-60hpoCk

**Comment by TheUnholyWrath** [comment]
> I am having exactly same problem like this can someone hlep me ?
> https://www.youtube.com/watch?v=jct-60hpoCk

unselect all pick and turbo, you can only queue single draft

**Comment by hansangwoo32** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/136219669/18fc79ae-cb29-4437-9294-5a7ec8917eed)
still not working.

**Comment by TheUnholyWrath** [comment]
> ![image](https://user-images.githubusercontent.com/136219669/244916762-18fc79ae-cb29-4437-9294-5a7ec8917eed.png)
> still not working.

unselect all pick. you can only queue single draft

**Comment by Aluerie** [comment]
> unselect **_all pick_** AND turbo

**Comment by hansangwoo32** [comment]
Thank to you all it's working now

**Comment by Skullfeier** [comment]
... i know i have to uncheck all but it dosnt show me all options to uncheck wtf am i suposed to do now .....

**Comment by Aluerie** [comment]
show screenshot or recording

**Comment by commentator24** [comment]
i was about to say what do you mean, then i realise how funny you say that because i cant see all options too lol
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/23db9f37-974e-4a29-9889-0fec52c3ef02)

this is an old post but might still work:
Rick-click Dota in Steam library, then select "set launch options"
Type -console, submit and then launch dota
In dota, type "\" to bring up console commands
type "dota_match_game_modes 16"
Should be able to find LP single draft match

**Comment by Skullfeier** [comment]
heyo thanks for the reply sadly it didn't work

**Comment by commentator24** [comment]
what do you mean it doesn't work? i just tested it and it changed my game mode to single draft only. 

as shown below i have everything i could see selected. Then when open console and typed game mode 16 it changed to single draft only after i click Play Dota again.

Before:
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/308e302c-2d1f-411f-8fc3-b48c547c10b2)

After:
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/2e33476a-d9dd-4df1-a790-d95c4de26adc)

what is the problem?
to enable console from launch option you go to steam library, right click property dota 2 and type -console
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/dd7f83cd-7d4d-4aec-bd4c-e41b4be9026a)

Or you can go to setting (advanced options) and enable it
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/42c46a76-1ce4-4339-9a58-838e08b17609)

To open console you set the hotkey in advanced options, right side. i use F8
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/ca22734f-5366-438f-8c03-1e9097792eb7)

if you still dont get it i don't know how else to help you aha. good luck
